pkg/apis/scaler/v1alpha1/types_demand.go

```diff
@@ -58,6 +58,7 @@ type DemandStatus struct {
 type DemandUnit struct {
 	CPU    resource.Quantity `json:"cpu"`
 	Memory resource.Quantity `json:"memory"`
+	GPU    resource.Quantity `json:"gpu,omitempty"`
```

Comment:
This API version is getting deprecated, I don't think we should be adding new things here.

Reply:
see above